components/org.wso2.carbon.identity.api.server.webhook.metadata/org.wso2.carbon.identity.api.server.webhook.metadata.v1/src/gen/java/org/wso2/carbon/identity/api/server/webhook/metadata/v1/WebhooksApi.java

```diff
@@ -99,14 +99,14 @@ public Response getEventProfiles() {
     @Path("/metadata")
     @Consumes({ "application/json" })
     @Produces({ "application/json" })
-    @ApiOperation(value = "Update webhook metadata properties", notes = "This API updates an webhook metadata property and return the updated webhook metadata properties.    <b>Scope (Permission) required:</b> ``internal_webhook_meta_update``  ", response = WebhookMetadataProperties.class, authorizations = {
+    @ApiOperation(value = "Update webhook metadata properties", notes = "This API updates an webhook metadata property and return the updated webhook metadata properties.    <b>Scope (Permission) required:</b> ``internal_webhook_meta_update``  ", response = WebhookMetadata.class, authorizations = {
```

Comment:
> This API updates an webhook metadata

---> This API updates a webhook metadata

Please check other occurrences as well.